srcalc/src/main/webapp/WEB-INF/views/fragments/booleanFragment.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```

Comment:
You should also have a comment specifying what attributes they expect to be defined.